src/locales/intl.test.js

```diff
@@ -0,0 +1,320 @@
+/* globals describe test expect beforeEach */
+import intl, {setIntlLocale} from './intl'
+import * as UTILS from '../../src/modules/utils'
```

Comment:
Please import specific routines. We are standardizing against `import *` going forward